ui/hooks/useAccountTotalFiatBalance.js

```diff
@@ -92,31 +91,26 @@ export const useAccountTotalFiatBalance = (
   };
 
   // To match the list of detected tokens with the entire token list to find the image for tokens
```

Comment:
The comment mentions images, but the icon URLs in the token list are always just `https://static.cx.metamask.io/api/v1/tokenIcons/${chainId}/${tokenAddress}.png` and could be calculated with `formatIconUrlWithProxy` instead. 

I had a previous PR [27781](https://github.com/MetaMask/metamask-extension/pull/27781) where I attempted to remove the token list lookup entirely, but `...matchingToken` was including fields other than the icon that I thought might break if removed, so I'm keeping it here.